src/Playwright/Transport/StdIOTransport.cs

```diff
@@ -70,9 +74,16 @@ public void Close(string closeReason)
         {
             if (!IsClosed)
             {
-                IsClosed = true;
-                TransportClosed?.Invoke(this, new() { CloseReason = closeReason });
-                _readerCancellationSource.Cancel();
+                try
+                {
+                    IsClosed = true;
+                    TransportClosed?.Invoke(this, new() { CloseReason = closeReason });
```

Comment:
We should not silent-catch code, especially code that invokes callbacks.

Reply:
Since it includes `process.kill` so had to enclose it in try/catch.
This is the current master code which does similar thing -> https://github.com/microsoft/playwright-dotnet/blob/f689cf4ec76b652c93946b183a46ab02222fd7d5/src/Playwright/Transport/Connection.cs#L449

Will only keep process object in try/catch others will remove out of it.

Reply:
fixed